static/src/js/components/CollectionSelector/CollectionSelector.jsx

```diff
@@ -0,0 +1,374 @@
+import React, {
+  useCallback,
+  useEffect,
+  useState
+} from 'react'
+
+import PropTypes from 'prop-types'
+
+import Badge from 'react-bootstrap/Badge'
+import Col from 'react-bootstrap/Col'
+import Form from 'react-bootstrap/Form'
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger'
+import Popover from 'react-bootstrap/Popover'
+import Row from 'react-bootstrap/Row'
+
+import {
+  FaMinus,
+  FaPlus,
+  FaTrash
+} from 'react-icons/fa'
+
+import { useLazyQuery, useSuspenseQuery } from '@apollo/client'
+
+import { GET_PERMISSION_COLLECTIONS } from '@/js/operations/queries/getPermissionCollections'
+
+import './CollectionSelector.scss'
```

Comment:
scss imports should be last, lodash-es import should be at the top with other 3rd party libraries